docs/model-dev-guide/batch-processing/batch-process-api-ug.rst

```diff
@@ -0,0 +1,173 @@
+.. _torch-batch-processing-ug:
+
+############################
+ Torch Batch Processing API
+############################
+
+.. meta::
+   :description: Learn how to use the Torch Batch Processing API.
+
+In this guide, you'll learn about the :ref:`Torch Batch Process API <torch-batch-process-api-ref>`
+and how to perform batch inference (also known as offline inference).
+
++---------------------------------------------------------------------+
+| Visit the API reference                                             |
++=====================================================================+
+| :ref:`torch-batch-process-api-ref`                                  |
++---------------------------------------------------------------------+
```

Comment:
i'm guessing this might need to be a link to the class header and not the title of the page

we would have to create an anchor for it 

e.g., ".. _pytorch-batch-process:"

Then change the :ref: to 

:ref:`pytorch-batch-process`

Reply:
we decided to link to the title of the page rather than a heading on the page.